proto/src/scim_v1/synch.rs

```diff
@@ -128,7 +133,7 @@ impl ScimSyncPerson {
                         SCIM_SCHEMA_SYNC_PERSON.to_string(),
                     ],
                     id,
-                    external_id: None,
+                    external_id: Some(external_id),
```

Comment:
Not 100% certain on whether we expect the `ScimSyncPerson` to be sent to us without `external_id` (or really anything scim related for that matter), But if not do we need the `external_id` field to still be `Option<String>`?

Reply:
SCIM rfc states that external ID is optional, but in our usage of it, it's required. That's why there is a disparity here.

Reply:
No stress, Just thought to ask because I wasn't sure. Good to go